tests/app/views/fixtures/conftest.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import absolute_import
+
+import pytest
+
+from app import create_app
+from app.models import db, utcnow, Supplier, SupplierDomain, User, Brief, \
+    Framework, Lot, Domain, Assessment
+from tests.app.helpers import COMPLETE_DIGITAL_SPECIALISTS_BRIEF, WSGIApplicationWithEnvironment
+
+from sqlbag import temporary_database
+from faker import Faker
+
+from migrations import \
+    load_from_app_model, load_test_fixtures
+
+fake = Faker()
+
+
+@pytest.fixture(autouse=True)
+def db_initialization(request):
+    from config import configs
+
+    with temporary_database() as dburi:
+        test_config = configs['test']
+        test_config.SQLALCHEMY_DATABASE_URI = dburi
+
+        load_from_app_model(dburi)
+        load_test_fixtures(dburi)
+
+        yield
+
+
+def setup_authorization(app):
+    valid_token = 'valid-token'
+    app.wsgi_app = WSGIApplicationWithEnvironment(
+        app.wsgi_app,
+        HTTP_AUTHORIZATION='Bearer {}'.format(valid_token))
+    app.config['DM_API_AUTH_TOKENS'] = valid_token
+
+
+@pytest.fixture()
+def app(request):
+    app = create_app('test')
+    app.config['SERVER_NAME'] = 'localhost'
+    setup_authorization(app)
+    yield app
+
+
+@pytest.fixture()
+def client(app):
+    yield app.test_client()
+
+
+@pytest.fixture()
+def suppliers(app, request):
+    with app.app_context():
+        for i in range(1, 6):
+            db.session.add(Supplier(
+                code=(i),
+                name=fake.name()
+            ))
+
+            db.session.flush()
+
+        db.session.commit()
+        yield Supplier.query.all()
+
+
+@pytest.fixture()
+def supplier_domains(app, request, suppliers):
+    with app.app_context():
+        for s in suppliers:
+            for i in range(1, 6):
+                db.session.add(SupplierDomain(
+                    supplier_id=s.id,
+                    domain_id=i
+                ))
+
+                db.session.flush()
+
+        db.session.commit()
+        yield SupplierDomain.query.all()
+
+
+@pytest.fixture()
+def users(app, request):
+    with app.app_context():
+        for i in range(1, 6):
+            db.session.add(User(
+                id=i,
+                email_address='{}@digital.gov.au'.format(fake.first_name()),
+                name=fake.name(),
```

Comment:
should we generate these with the same seed each time so we get deterministic names generated?